rhobs/alerting/data_plane/prometheus.serviceprovider_alerts.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-spi-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: serviceprovider_alerts
+      interval: 1m
+      rules:
+        - alert: ServiceProvider5xxErrorsRate
+          expr: |
+            (
+              sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count{status=~"5.."}[1h]))
+              or sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count{}[1h])) * 0
```

Comment:
Can drop the empty `{}`.

Reply:
We should have a test case in which the RHS of the `or` manifests. i.e. a test in which the LHS returns no time series.

Reply:
Added